thread.h

```diff
@@ -379,19 +379,39 @@
 #    define PTHREAD_GETSPECIFIC(key) pthread_getspecific(key)
 #endif
 
-#ifndef PERL_GET_CONTEXT
-#  define PERL_GET_CONTEXT	PTHREAD_GETSPECIFIC(PL_thr_key)
-#endif
+#if defined(PERL_THREAD_LOCAL) && !defined(PERL_GET_CONTEXT) && !defined(PERL_GET_CONTEXT)
```

Comment:
Duplicate `&& !defined(PERL_GET_CONTEXT)`?

Reply:
Oops yes. One should be PERL_SET_CONTEXT.

Reply:
Pushing an updated version with this fixed.